lib/charty/colors/color_data.rb

```diff
@@ -0,0 +1,1148 @@
+module Charty
+  module Colors
+    module ColorData
+      # The default value of matplotlib's rcParams['axes.prop_cycle'].by_key().get('color')
+      DEFAULT_COLOR_CYCLE = [ -'#1f77b4', -'#ff7f0e', -'#2ca02c', -'#d62728', -'#9467bd',
+                              -'#8c564b', -'#e377c2', -'#7f7f7f', -'#bcbd22', -'#17becf' ].freeze
+
+      # Matplotlib's base colors
+      BASE_COLORS = {
+        "b" => [0r,    0r,    1r].freeze,
+        "g" => [0r,    0.5r,  0r].freeze,
+        "r" => [1r,    0r,    0r].freeze,
+        "c" => [0r,    0.75r, 0.75r].freeze,
+        "m" => [0.75r, 0,     0.75r].freeze,
+        "y" => [0.75r, 0.75r, 0r].freeze,
+        "k" => [0r,    0r,    0r].freeze,
+        "w" => [1r,    1r,    1r].freeze
+      }.freeze
+
+      # Colors from Tableau
+      TABLEAU_COLORS = {
+        "tab:blue"   => -"#1f77b4",
+        "tab:orange" => -"#ff7f0e",
+        "tab:green"  => -"#2ca02c",
+        "tab:red"    => -"#d62728",
+        "tab:purple" => -"#9467bd",
+        "tab:brown"  => -"#8c564b",
+        "tab:pink"   => -"#e377c2",
+        "tab:gray"   => -"#7f7f7f",
+        "tab:olive"  => -"#bcbd22",
+        "tab:cyan"   => -"#17becf"
+      }.freeze
+
+      # This mapping of color names -> hex values is taken from
+      # a survey run by Randall Munroe see:
+      # https://blog.xkcd.com/2010/05/03/color-survey-results/
+      # for more details.  The results are hosted at
+      # https://xkcd.com/color/rgb
+      # and also available as a text file at
+      # https://xkcd.com/color/rgb.txt
+      #
+      # License: http://creativecommons.org/publicdomain/zero/1.0/
```

Comment:
How about adding a script or Rake task to generate these code to update these code?

Reply:
I filed #41.